CONTRIBUTING.md

```diff
@@ -61,8 +61,9 @@ To add support for a new language:
   This command will analyze the grammar and, if no problems are found, add it to the repository. If problems are found, please report them to the grammar maintainer as you will not be able to add the grammar if problems are found.  
   **Please only add grammars that have [one of these licenses][licenses].**
 1. Add samples for your language to the [samples directory][samples] in the correct subdirectory.
-1. Add a `language_id` for your language using `script/set-language-ids`.  
-  **You should only ever need to run `script/set-language-ids --update`. Anything other than this risks breaking GitHub search :cry:**
+1. Add a `language_id` for your language using `script/set-language-ids --update`.  
+  **You should only ever need to run with `--update`. Anything other than this risks breaking GitHub search :cry:**
+1. Add an entry to the [vendor README](vendor/README.md) using `script/list-grammars`.
```

Comment:
This step shouldn't be necessary; `script/add-grammar` already calls `script/list-grammar`.

Reply:
Hmm, I've always had to run `list-grammar` manually after adding or else vendor/README.md isn't updated and the tests fail (e.g. https://travis-ci.org/github/linguist/builds/405880762)

Reply:
@wesinator Could you file a separate issue for that with example steps to reproduce?